Assets/Scripts/Gameplay/GameState/NetworkWinState.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom
+{
+    public enum WinState
+    {
+        Invalid,
+        Win,
+        Loss
+    }
+
+    /// <summary>
+    /// MonoBehaviour containing only one NetworkVariableBool to represent the game session's win state.
+    /// </summary>
+    public class NetworkWinState : NetworkBehaviour
```

Comment:
"PersistentGameState" for the name? We already have that pattern with persistent player vs avatar for cross scene objects?